### PR TITLE
Update instruction for new project creation

### PR DESCRIPTION
**Pull request by e-backmark-ericsson**
### Applicable Issues
#123 

### Description of the Change
Updates the current instruction on how to create a new project in the Eiffel Community GitHub organization

### Alternate Designs
-

### Benefits
Clarity

### Possible Drawbacks
None

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Emil Bäckmark, emil.backmark@ericsson.com

### CODE REVIEW

File: PROJECT_LIFECYCLE.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ mandatory.
 
 Simple majority of Eiffel Community Maintainers is required for a project to be accepted to Eiffel Community.
 
-Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) by sending a pull request to community
-repository and include an entry for their project. Project creation is concluded upon acceptance of the pull
-request.
+Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) and [CONTACTS.md](./CONTACTS.md) by sending a pull request to community
+repository and include an entry for their project. Project acceptance is concluded upon acceptance of the pull
+request. When accepted, Eiffel Community organization owners create the GitHub project in accordance with the [repository checklist](https://github.com/eiffel-community/eiffel-repository-template/blob/master/repo-checklist.md) and merge the corresponding pull request.
```
Comment (magnusbaeck) [inline PROJECT_LIFECYCLE.md:65]: This appears to be a change in the current process. Is there any reason to not let the repo proposer drive the process?

**Review by magnusbaeck — COMMENTED** [review]
Change description to say "Fixes #123" or do you expect additional commits?

File: PROJECT_LIFECYCLE.md (hunk 1 above)
Comment (e-backmark-ericsson) [reply PROJECT_LIFECYCLE.md:65]: I'm under the impression that only org owners are able to create new repos, isn't it so?

Comment (magnusbaeck) [reply PROJECT_LIFECYCLE.md:65]: As I interpret https://github.com/organizations/eiffel-community/settings/member_privileges any organization member can create repos.

Comment (e-backmark-ericsson) [reply PROJECT_LIFECYCLE.md:65]: Great! All the better :) I'll update the instruction.

File: PROJECT_LIFECYCLE.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ mandatory.
 
 Simple majority of Eiffel Community Maintainers is required for a project to be accepted to Eiffel Community.
 
-Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) by sending a pull request to community
-repository and include an entry for their project. Project creation is concluded upon acceptance of the pull
-request.
+Project maintainers are expected to update [PROJECTS.md](./PROJECTS.md) and [CONTACTS.md](./CONTACTS.md) by sending a pull request to community
+repository and include an entry for their project. Project acceptance is concluded upon acceptance of the pull
+request. When accepted, the maintainers of the accepted project should create the repository in accordance with the [repository checklist](https://github.com/eiffel-community/eiffel-repository-template/blob/master/repo-checklist.md) and then merge the corresponding pull request.
```
Comment (magnusbaeck) [inline PROJECT_LIFECYCLE.md:65]: > and then merge the corresponding pull request

But _that_ needs to be done by a community maintainer.

Comment (e-backmark-ericsson) [reply PROJECT_LIFECYCLE.md:65]: Good to clarify that then ;)

**Review by magnusbaeck — APPROVED** [review]

**Review by t-persson — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Pull request merged**